docs/Contributors_Guide/continuous_integration.rst

```diff
@@ -151,3 +431,524 @@ for the step named "Get METplus Image."
+      }
+
+This example defines a use case group that contains the climate use case
+with index 2 and is marked to "run" for every push.
+
+
+.. _cg-ci-subset_category:
+
+Subset Category into Multiple Tests
+"""""""""""""""""""""""""""""""""""
+
+Use cases can be separated into multiple test jobs.
+In the "index_list" value, define the cases to run for the job.
+Use cases are numbered starting with 0 and are in order of how they are
+found in the all_use_cases.txt file.
+
+The argument supports a comma-separated list of numbers. Example::
+
+      {
+        "category": "data_assimilation",
+        "index_list": "0,2,4",
+        "run": false
+      },
+      {
+        "category": "data_assimilation",
+        "index_list": "1,3",
+        "run": false
+      },
+
+The above example will run a job with data_assimilation use cases 0, 2, and
+4, then another job with data_assimilation use cases 1 and 3.
+
+It also supports a range of numbers separated with a dash. Example::
+
+      {
+        "category": "data_assimilation",
+        "index_list": "0-3",
+        "run": false
+      },
+      {
+        "category": "data_assimilation",
+        "index_list": "4-5",
+        "run": false
+      },
+
+The above example will run a job with data_assimilation 0, 1, 2, and 3, then
+another job with data_assimilation 4 and 5.
+
+You can also use a combination of commas and dashes to define the list of cases
+to run. Example::
+
+      {
+        "category": "data_assimilation",
+        "index_list": "0-2,4",
+        "run": false
+      },
+      {
+        "category": "data_assimilation",
+        "index_list": "3",
+        "run": false
+      },
+
+The above example will run data_assimilation 0, 1, 2, and 4 in one
+job, then data_assimilation 3 in another job.
+
+Run Use Cases
+^^^^^^^^^^^^^
+
+The **use_case_tests** job is duplicated for each use case group using the
+strategy -> matrix syntax::
+
+    strategy:
+        fail-fast: false
+        matrix: ${{fromJson(needs.job_control.outputs.matrix)}}
+
+**fail-fast** is set to false so that the rest of the use case test jobs will
+run even when one of them fails. The **matrix** value is a list of use
+case categories and indices that is created in the :ref:`cg-ci-job-control`
+job. Each value in the list is referenced in the job steps with
+**${{ matrix.categories }}**::
+
+    - name: Run Use Cases
+      uses: ./.github/actions/run_tests
+      id: run_tests
+      with:
+        categories: ${{ matrix.categories }}
+
+The logic that runs the use cases is contained in a custom GitHub Action
+that is found in the METplus repository.
+
+Obtaining Input Data
+""""""""""""""""""""
+
+Each use case category has a corresponding Docker data volume that contains
+the input data needed to run all of the use cases. The data volume is obtained
+from DockerHub and mounted into the container that will run the use cases
+using the **--volumes-from** argument to the **docker run** command.
```

Comment:
The --volumes-from is NOT being rendered correctly via RTD. See this [section](https://metplus.readthedocs.io/en/feature_675_ci_doc/Contributors_Guide/continuous_integration.html#obtaining-input-data).